scripts/install

```diff
@@ -115,7 +113,7 @@ ynh_script_progression --message="Configuring system user..." --weight=1
 # Add the user to the signald group. The signald group was created when the signald
 # package was installed from the extra repository
 # resolved by https://gitlab.com/signald/signald/-/commit/278240f3f1cc40a3b444c958b68ca3d6908e98a8
-ynh_system_user_create --username=$app --groups="$signald_user"
+ynh_system_user_create --username=$app
```

Comment:
Why remove signald group membership? This will cause permissions issues when accessing the socket.

If that's needed, also remove the comment l.113

Reply:
> As far as I understood it was solved upstream. But it was not tested thoroughly yet. That's why I would keep these comments for now. We can remove it in a later release.

@Gredin67, I haven't seen anything like that upstream, but if it is working on a first install, it should be fine.

Reply:
> As far as I understood it was solved upstream. But it was not tested thoroughly yet. That's why I would keep these comments for now. We can remove it in a later release.

@Gredin67, I haven't seen anything like that upstream, but if it is working on a first install, it should be fine.

Reply:
Thanks for testing!